hack/update-translations.sh

```diff
@@ -81,12 +82,10 @@ fi
 
 if [[ "${generate_mo}" == "true" ]]; then
   echo "Generating .po and .mo files"
-  for x in translations/*/*/*/*.po; do
+  for x in "${TRANSLATIONS}"/*/*/*/*.po; do
```

Comment:
if we drop the `kubectl` subdir, would change to:

```suggestion
  for x in "${TRANSLATIONS}"/*/*/*.po; do
```